### DISCUSSION TITLE
Hide script from main list?

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1464] [Q&A] [answered]
Hey! long time now see
Question if someone knows: Any way of hiding or de-prioritizing scripts from the main list? For example Scripts which sole purpose is to schedule something periodically, I don't need to run it manually. If I just modify the script it will go back on top of the Recent list :P

**Comment by JosXa** [comment]
Perhaps you can trick it by calling `setScriptTimestamp` after every execution? 🤔 
```ts
export declare let setScriptTimestamp: (stamp: Stamp) => Promise<Script[]>;
export type Stamp = {
    filePath: string;
    timestamp?: number;
    compileStamp?: number;
    compileMessage?: string;
    executionTime?: number;
    changeStamp?: number;
    exitStamp?: number;
    runStamp?: number;
    runCount?: number;
};
```

**Reply by johnlindquist** [reply, marked as answer]
@ramiroaraujo Adding:

```js
// Exclude: true
```

Will keep it from appearing in the main menu

**Reply by JosXa** [reply]
Docs where? 😅
